README.md

```diff
@@ -195,3 +213,10 @@ spec:
 - [No Log](./docs/user-guide/advanced-configuration/no-log.md)
 - [Deploy a Specific Version of EDA](./docs/user-guide/advanced-configuration/deploying-a-specific-version.md)
 - [Trusting a Custom Certificate Authority](./docs/user-guide/advanced-configuration/trusting-a-custom-certificate-authority.md)
+
+## Maintainers Docs
+
+Maintainers of this repo need to carry out releases, triage issues, etc. There are docs for those types of administrative tasks in the `docs/maintainer/` directory.
```

Comment:
Food for thought, maintainer and contributing docs be moved to a central location.

Thought Process. I know of at least three repos in the ansible org where the structures and processes appear the same:

- awx
- awx-operator
- eda-operator

A single location for common items give a single point for them to be updated and referenced by all. Normalizing the processes, i.e Actions, Workflows, Contributing, Dev etc. Reduces the burden on those who work on the repos as now they need to learn one process to work on "Ansible" repos and the process they learn is portable to all repos. An update to the source docs, should add a workflow item to review and refactor a repos "processes" to be in line and ensures it remains up to date and inline with best practices.

Reply:
@jon-nfc I :100: agree.  We an internal repo where we could potentially put standard release docs like that, but no public equivalent.  I really want to keep these docs public.  

A step in the right direction would be to have a single set of docs for Actions, Workflows, Contributing topics in one of those mentioned repos (maybe awx?), then to link to that from the other repo's readme's.  However, having a central place would be better, as a landing page for people interested in contributed to any of the Ansible related repos.  

The developer docs will still be largely different for each repo, as developing for say ansible-core, will be vastly different then awx, or any of the operators.  But I could certainly see a single development.md doc for all of the operators.  

There is leg work needed to standardize these processes in each operator repo, and arguably more to be done in the application repos (awx, eda, galaxy_ng), but it is definitely worth the time.  I think we should capture this in a separate issue.